plugins/cc/cctac.ml

```diff
@@ -583,3 +583,8 @@ let f_equal =
         | e -> Proofview.tclZERO ~info e
       end
   end
+
+(* we need to be registered in case we are statically linked to avoid double loading
+   (in practice we are statically linked in coqtop.byte:
+    coqtop.byte -> coq-core.dev -> ltac2 -> cc_core) *)
+let () = Mltop.add_known_module "cc_core_plugin:coq-core.plugins.cc_core"
```

Comment:
This is not a very nice way to prevent double loading but I couldn't come up with something better.
Trying to check Dynlink.all_units fails in coqtop.byte because "The dynlink.cma library cannot be used inside the OCaml toplevel" (how the dynlink itself works IDK).
I don't know how else to find out what may be statically linked.

Reply:
This should have been added by `coqpp`, no?

Reply:
no coqpp files in cc_core